Makefile

```diff
@@ -220,6 +229,43 @@ ifndef OPERATOR_SECRET_ACCESS_KEY
 	$(error OPERATOR_SECRET_ACCESS_KEY is undefined)
 endif
 
-# Test all
+# Test apply ou map cr
+.PHONY: create-ou-map
+create-ou-map:
+	# Create OU map
+	@oc process -p ROOT=${OSD_STAGING_1_OU_ROOT_ID} -p BASE=${OSD_STAGING_1_OU_BASE_ID} -f hack/templates/aws_v1alpha1_configmap.tmpl | oc apply -f -
+
+# Test delete ou map cr
+.PHONY: delete-ou-map
+delete-ou-map:
+# Delete OU map
+	@oc process -p ROOT=${OSD_STAGING_1_OU_ROOT_ID} -p BASE=${OSD_STAGING_1_OU_BASE_ID} -f hack/templates/aws_v1alpha1_configmap.tmpl | oc delete -f -
+
+# Test claim-account
+.PHONE: claim-ou-account
+claim-ou-account:
```

Comment:
Would it make more sense for this to be
`claim-ou-account: create-accoute create-account-claim-namespace create-account-claim` ?
 That way the logic of each of those doesn't have to be repeated

Reply:
Since targets can only be used a single time per run I tried to keep these targets as self contained as possible so that we can add it to the `test-all` target.

Reply:
Why not just test add OU stuff as part of normal account creation/claiming then?

Reply:
Done

Reply:
I don't see where this is done. I am suggesting to combine this ou testing with the current account/accountclaim targets

Reply:
sorry, I forgot to remove this target.